archunit/src/main/java/com/tngtech/archunit/library/Architectures.java

```diff
@@ -167,9 +175,65 @@ public EvaluationResult evaluate(JavaClasses classes) {
             for (LayerDependencySpecification specification : dependencySpecifications) {
                 result.add(evaluateDependenciesShouldBeSatisfied(classes, specification));
             }
+            if (allClassesAreCovered.isPresent()) {
+                result.add(allClassesAreCovered.get().evaluate(classes, layerDefinitions));
+            }
             return result;
         }
 
+        @PublicAPI(usage = ACCESS)
+        public LayeredArchitecture whereAllClassesAreCovered() {
```

Comment:
I'm not sure if the expression "covered" is in general easy to understand here. Maybe `whereAllClassesAreContainedInLayers`? Just thinking that "covered" is used nowhere else in ArchUnit, as far as I remember, but "contained" is used.
Or even name it `ensureAllClassesAreContainedInLayers` or similar, since all the `where` clauses are part of the specification, i.e. "this is how our layered architecture is defined", while this here is more like a configuration how this layered architecture should be tested against the classes supplied.

Reply:
Fair enough. `ensureAllClassesAreContainedInLayers` and similar seem very verbose and so I was shooting for something more succinct - but apparently lost clarity in the process. Also hadn't picked up on the `where` prefix suggesting the spec rather than a test. Will rework based on your naming.

Reply:
Done in a9740ba.